deployment/base/scripts/init_content/03.EventNotificationTemplate.0.template.xml

```diff
@@ -1015,5 +1015,754 @@
 		</request>
 	</multirequest>
 	
+	<multirequest>
+		<request service="eventNotificationTemplate" action="add" plugin="eventNotification" partnerId="0">
+			<template objectType="KalturaHttpNotificationTemplate">
+				<name>Entry Status Changed</name>
+				<systemName>HTTP_ENTRY_STATUS_CHANGED</systemName>
+				<description>HTTP notification template to be sent when an entry status changed.</description>
+				<automaticDispatchEnabled>1</automaticDispatchEnabled>
+				<eventType>3</eventType> <!-- EventNotificationEventType::OBJECT_CHANGED -->
+				<eventObjectType>1</eventObjectType> <!-- EventNotificationEventObjectType::ENTRY -->
+				<eventConditions objectType="array">
+					<item objectType="KalturaEventFieldCondition">
+						<description>Status field modified</description>
+						<field objectType="KalturaEvalBooleanField">
+							<code>in_array(entryPeer::STATUS, $scope->getEvent()->getModifiedColumns())</code>
+						</field>
+					</item>
+				</eventConditions>
+				<contentParameters objectType="array">
+					<item objectType="KalturaEventNotificationParameter">
+						<key>entry_id</key>
+						<description>Entry ID</description>
+						<value objectType="KalturaEvalStringField">
+							<code>$scope->getEvent()->getObject()->getId()</code>
```

Comment:
Please use $scope->getObject() instead of $scope->getEvent()->getObject() in all places, it will enable manual dispatching in the future.